src/components/molecules/TableGrid/index.ts

```diff
@@ -0,0 +1 @@
+export { TableGrid } from "./TableGrid";
```

Comment:
Should probably delete `src/components/molecules/TablesUserList/TablesUserList.scss`, not just leave empty